examples/simple-game-server/Makefile

```diff
@@ -25,13 +25,18 @@
 
 REGISTRY = gcr.io/agones-images
 BUILDX_WINDOWS_BUILDER = windows-builder
-WINDOWS_VERSIONS = ltsc2019 # 1909 (For GKE WINDOWS_SAC support)
+# Versions of Windows Server to support, default is Windows Server 2019.
+# For the full list of tags see https://hub.docker.com/_/microsoft-windows-servercore.
+# GKE-Windows version map: https://cloud.google.com/kubernetes-engine/docs/how-to/creating-a-cluster-windows#version_mapping
+WINDOWS_VERSIONS = ltsc2019
 WINDOWS_DOCKER_PUSH_ARGS = # When pushing set to --push.
+# Build with Windows support
+WITH_WINDOWS=1
 
 mkfile_path := $(abspath $(lastword $(MAKEFILE_LIST)))
 project_path := $(dir $(mkfile_path))
 server_tag = $(REGISTRY)/simple-game-server:0.2
-ifdef WITH_WINDOWS
+ifeq ($(WITH_WINDOWS), 1)
```

Comment:
Just so I'm clear, this means that on the next release, the section "Images available with this release:" (e.g. [here](https://agones.dev/site/blog/2021/03/16/1.13.0-kubernetes-1.18-websocket-watch-sdk-empty-ports/)) will include: `gcr.io/agones-images/simple-game-server:0.2-linux_amd64` and `gcr.io/agones-images/simple-game-server:0.2-windows_amd64-ltsc2019` and `gcr.io/agones-images/simple-game-server:0.2` (where the last one will now be a multi-arch image. Is that correct?

For the multi-arch image, does pulling it use more disk space (pulling the windows & linux versions) or is docker smart enough to only pull for your architecture (I think the latter is true but haven't checked in a long time)? I want to make sure the release notes explain how this will affect users. 

The same question applies to the sdk sidecar image (above) since that is the other image that we will now be publishing this same way.

Reply:
Windows Server 1909 will stop working on May 11, 2021. It's not productive to support it. The builds will not build Windows SAC flavors of the image only LTSC.
When docker queries an image registry if an image manifest is returned it'll select the most appropriate image based on the host's characteristics. I believe there's an API to basically have select the correct image for you without a second request. as well.

Additional space will be used for each version of the image but the windows base layers are stored on mcr.microsoft.com so Windows Server Core is not stored in gcr.io. Only the layers on top this is due to the foreign layers bit Microsoft sets on these images.

Reply:
I think it'll be better just to remove references to Windows SAC altogether. It appears to be adding confusion.